pkg/util/util.go

```diff
@@ -0,0 +1,39 @@
+package util
+
+import (
+	"os"
+
+	logf "github.com/openshift/router/log"
+)
+
+var log = logf.Logger.WithName("util")
+
+// CopyFile copies a file from src to dest. It returns an error on failure
+func CopyFile(src, dest string) error {
+	in, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err := in.Close(); err != nil {
+			log.Error(err, "Failed to close input file", "filename", src)
+		}
+	}()
+	out, err := os.Create(dest)
+	if err != nil {
+		return err
+	}
+	_, err = out.ReadFrom(in)
+	if err != nil {
+		if err := out.Close(); err != nil {
+			log.Error(err, "Failed to close output file", "filename", dest)
+			return err
+		}
+		return err
```

Comment:
I think you want to log the error from `Close` and return the error from `ReadFrom`, right?  No need to log _and_ return the same error (which in this case will ultimately result in the error getting logged twice when the caller logs the returned error).  
```suggestion
		if err := out.Close(); err != nil {
			log.Error(err, "Failed to close output file", "filename", dest)
		}
		return err
```